eng/common/scripts/Save-Package-Properties.ps1

```diff
@@ -64,6 +64,10 @@ function SetOutput($outputPath, $incomingPackageSpec) {
 }
 
 function GetRelativePath($path) {
+  # If the path is empty return an empty string
```

Comment:
Do we have a case where we are validly passing empty here?

Reply:
In this instance `Azure.Core.TestFramework` does not have a `CHANGELOG.md` https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/core/Azure.Core.TestFramework 

In [previous builds](https://dev.azure.com/azure-sdk/public/_build/results?buildId=974443&view=artifacts&pathAsName=false&type=publishedArtifacts) this would have produced output like: 

```json
{
  "Name": "Azure.Core.TestFramework",
  "Version": "",
  "DirectoryPath": "D:\\a\\_work\\1\\s\\sdk\\core\\Azure.Core.TestFramework",
  "ServiceDirectory": "core",
  "ReadMePath": "D:\\a\\_work\\1\\s\\sdk\\core\\Azure.Core.TestFramework\\README.md",
  "ChangeLogPath": "",
  "Group": null,
  "SdkType": "client",
  "IsNewSdk": true,
  "ArtifactName": "Azure.Core.TestFramework",
  "ReleaseStatus": null
}
```